app/src/main/res/values-v19/themes.xml

```diff
@@ -3,6 +3,5 @@
     <style name="AppTheme.NoActionBar.Splash" parent="AppTheme.NoActionBar">
         <item name="android:windowTranslucentStatus">true</item>
         <item name="android:windowTranslucentNavigation">true</item>
-        <item name="android:background">@drawable/bg_splash</item>
```

Comment:
this option change ImageView's background too, so i remove.
Instead I set this to splash's root-view-group.